src/software/ai/hl/stp/play_info.h

```diff
@@ -3,22 +3,57 @@
 #include <string>
 #include <vector>
 
-struct PlayInfo
+class PlayInfo
```

Comment:
We're missing a class comment here to explain what PlayInfo is and why it's useful

Reply:
Tbh, I am not quite sure I completely get what the class does. However, made an attempt to explain it based on its use cases.  Please double check!

Reply:
I believe it's just used for the visualization of what's happening on the field. We like to know which play is running and which robots are assigned each tactic.

Reply:
I believe this is correct. We shouldn't be documenting this sort of "upstream" dependency in the jdoc though, it should just describe _what_ a class is, maybe how it should be used (in the case of this class, there's not a ton to explain).

Reply:
Sorry that should have been added by me originally. You can add something like

"PlayInfo represents information about a play / strategy, including the name of the play, the gamestate the play is running in, and what tactics the play is using."

Feedback welcome.

Reply:
'A description of the play.  Contains the type of play being run, its name, and the tactic assignments for all the robots on the field.' This was in my last commit.  Similar enough?
@MathewMacDougall

Reply:
:+1: